tools/cmake/executorch-config.cmake

```diff
@@ -73,20 +73,23 @@ set(backend_lib_list
     vulkan_backend
 )
 
-set(lib_list
-    flatccrt
-    etdump
-    bundled_program
+set(extension_lib_list
```

Comment:
I think we don't have to do this after EXPORT is enabled. cc @swolchok

Reply:
executorch_extensions isn't a target in our internal builds, so we'll still need to create it as we do here because there's nothing to export.

Reply:
> executorch_extensions isn't a target in our internal builds

Sorry, this was totally wrong, I somehow completely missed the CMakeLists.txt changes.